packages/react-core/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/Icon/icon';
+import { css } from '@patternfly/react-styles';
+import { Spinner } from '../Spinner';
+
+export interface IconComponentProps extends Omit<React.HTMLProps<HTMLSpanElement>, 'size'> {
+  /** Icon content when isInProgress is set to false */
+  children?: React.ReactNode;
+  /** Icon content when isInProgress is set to true. Defaults to a medium spinner with 1em diameter.  */
```

Comment:
```suggestion
  /** Icon when isInProgress is set to true. Defaults to a 1em spinner.  */
```

I'm not sure if you need the bit about the spinner size, but if you do, we should only mention the diameter since the size class will be overridden. In our next sprint, I'd like to add the "inline" spinner and get away from having to set a custom diameter, too.